packages/vuetify/src/components/VSlider/VSlider.ts

```diff
@@ -549,7 +553,8 @@ export default mixins<options &
       if ([left, right, down, up].includes(e.keyCode)) {
         this.keyPressed += 1
 
-        const increase = this.$vuetify.rtl ? [left, up] : [right, up]
+        const increase = this.decreasing ? this.vertical ? [right, down] : [left, up]
```

Comment:
that's a bit unreadable, also i wonder if we should handle left/right for vertical slider?

Reply:
will try to make it more readable